### ISSUE TITLE
Update npm packages to get rid of vulnerabilities

### ISSUE THREAD

**Issue by giovanecosta** [HathorNetwork/hathor-explorer#59]
Running `npm install` on project give us the following message:
![image](https://user-images.githubusercontent.com/698586/119668121-449e2800-be0d-11eb-83e2-541e81df3795.png)

Not sure it will be easy to upgrade, but i think we have to do it asap.

**Comment by D45putspin** [comment]
solved the majority of them with `npm audit fix --force `
 and `npm update`
[](url)
![Captura de ecrã 2021-05-26, às 14 49 44](https://user-images.githubusercontent.com/36547913/119671466-aff9f100-be31-11eb-8a72-38614046b7e1.png)

**Comment by pedroferreira1** [comment]
> solved the majority of them with `npm audit fix --force `
> and `npm update`
> 
> ![Captura de ecrã 2021-05-26, às 14 49 44](https://user-images.githubusercontent.com/36547913/119671466-aff9f100-be31-11eb-8a72-38614046b7e1.png)

Yeah, thanks! We just need to run our QAs and check if we keep compatibility with all new package versions.
